### ISSUE TITLE
[BUG] Bright colors use the same sequences as regular colors on Command Prompt

### ISSUE THREAD

**Issue by jack1142** [willmcgugan/rich#938]
**Describe the bug**
On Command Prompt, bright colors use the same escape sequences as regular colors (e.g. `\x1b[30m`) rather than the ones for bright colors (e.g. `\x1b[90m`).
I would expect bright colors to either use the bold escape sequence or just use the regular bright color sequence, either works the same in Command Prompt so it doesn't really matter which of these would be used.

**To Reproduce**
```py
import rich

def show_color(x):
    c.print(f"[color({x})]TEXT[/]    [color({x+8})]TEXT[/]    [color({x}) bold]TEXT[/]")

c = rich.get_console()

c.print("normal  bright  bold\n")
for x in range(8):
    show_color(x)

c.print("\nnormal                 bright                 bold\n")
with c.capture() as capture:
    for x in range(8):
        show_color(x)
print(capture.get().replace("\x1b", "\\x1b"))
```
Command Prompt:
![image](https://user-images.githubusercontent.com/6032823/104922800-77d23580-599b-11eb-8280-5fad913aa27d.png)
Windows Terminal for comparison:
![image](https://user-images.githubusercontent.com/6032823/104922897-933d4080-599b-11eb-9d54-2a48b3f58b25.png)

**Platform**
Windows with Command Prompt and for comparison Windows with Windows Terminal.

**Diagnose**
Command Prompt:
![image](https://user-images.githubusercontent.com/6032823/104923115-e7482500-599b-11eb-9aaf-63068bd23c46.png)
![image](https://user-images.githubusercontent.com/6032823/104923172-faf38b80-599b-11eb-8f4a-9ae0243f0207.png)
Windows Terminal for comparison:
![image](https://user-images.githubusercontent.com/6032823/104923259-19f21d80-599c-11eb-97d9-42a1ab7987f7.png)
![image](https://user-images.githubusercontent.com/6032823/104923286-22e2ef00-599c-11eb-9422-d7c8212e4ae0.png)

Running Rich 9.8.2 on both.

**Comment by jack1142** [comment]
From what I gathered, changing few methods in `Color` to support 16 rather than 8 colors would be enough to fix it on Windows 10, but I've got no clue if that won't break on older Windows versions that don't support ANSI escape sequences.
So if that were problematic, it could be better to instead use the standard color system on Windows versions that support ANSI escape sequences, but I don't really know whether that would be easy to accomplish.

**Comment by willmcgugan** [comment]
The problem with bright colors on Windows is bold would have no effect, i.e. "bold bright_red" renders the same as "bright_red" - since true bold isn't supported. That and the fact that the default color choices for Windows is terrible. Some of the "bright" versions are barely distinguishable from the regular versions.

That was my thinking at the time. I'm not a windows user myself. If you think that it is better to explicitly expose all 16 colors I'm happy to make the change.

**Comment by jack1142** [comment]
You're right that bold has the same effect as using the bright color and that "bolded" bright color and not "bolded" will be the same on Windows. But even with that said, I think it's easier to make a multiplatform application *if* you can still differ between bright color and bold color on platforms that do support it. This way, you can at least still expect the "bright" colors to be "bright" on *all* platforms, rather than use bold just so that it can also appear "bright" on Windows.

So to summarize, I think that exposing all 16 colors on Windows would indeed work better.

**Comment by willmcgugan** [comment]
v9.9.0 exposes all 16 colors.
